openedx/core/lib/xblock_utils.py

```diff
@@ -302,7 +302,9 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
         # check that the course is a mongo backed Studio course before doing work
         is_studio_course = block.course_edit_method == "Studio"
 
-        if is_studio_course:
+        is_ccx_course = hasattr(course_id, 'ccx')
+        # preventing users from opening ccx courses in Studio, since Studio doesn't support CCX courses
+        if is_studio_course and not is_ccx_course:
             # build edit link to unit in CMS. Can't use reverse here as lms doesn't load cms's urls.py
             edit_link = "//" + settings.CMS_BASE + '/container/' + unicode(block.location)
```

Comment:
Hi @ormsbee, I don't know if I got it. What you're talking it's a proposal or I should follow your ideas to get this PR merged?

PS: I don't have a deep knowledge of the project, so I'll take a few days to get this done (if the change on my code is necessary).

tks